src/cmd_argument.rs

```diff
@@ -232,3 +232,33 @@ where
         config.working_directory = Some(self.0.as_ref().to_owned());
     }
 }
+
+/// Arguments of type [`PathBuf`] are passed into the child process
+/// as arguments.
+///
+/// ```
+/// use cradle::*;
+/// use std::path::PathBuf;
+///
+/// let current_dir: PathBuf = std::env::current_dir().unwrap();
+/// cmd_unit!("ls", current_dir);
+/// ```
+impl CmdArgument for PathBuf {
+    #[doc(hidden)]
+    fn prepare_config(self, config: &mut Config) {
+        config.arguments.push(self.into_os_string());
+    }
+}
+
+/// Arguments of type [`&Path`] are passed into the child process
```

Comment:
```suggestion
/// Arguments of type [`&Path`] are passed to the child process
```

Reply:
Done in https://github.com/soenkehahn/cradle/pull/60/commits/1d5014d50b1e1e35fe0db03eeebf18610bdd6a16.